wire-library/wire-gradle-plugin/src/main/kotlin/com/squareup/wire/gradle/WirePlugin.kt

```diff
@@ -109,7 +109,10 @@ class WirePlugin : Plugin<Project> {
       }
     }
 
-    val outputs = extension.outputs.ifEmpty { listOf(JavaOutput()) }
+    val outputs = extension.outputs
+    check(outputs.isNotEmpty()) {
+      "At least one target must be provided for project '${project.path}"
```

Comment:
I wonder if we're gonna get a lot of failing consumers with this.
How about we link our doc https://square.github.io/wire/wire_compiler/#customizing-output
What do you think? I'm not sure it's obvious what a target is.


```suggestion
      "At least one target must be provided for project '${project.path}\n" +
          "See our documentation for details: https://square.github.io/wire/wire_compiler/#customizing-output"
```

Reply:
I think that is a great idea :fire